/Comparison:
  ros__parameters:
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: best_effort
    use_sim_time: false
    pub_topic: transfer_topic
    sub_topic: velodyne_points
